services/supervisord/logs.go

```diff
@@ -153,16 +153,13 @@ func (l *Logs) files(ctx context.Context) []fileContent {
 		"/etc/nginx/conf.d/pmm-ssl.conf",
 
 		"/srv/alertmanager/alertmanager.base.yml",
-		"/srv/prometheus/prometheus.base.yml",
```

Comment:
wow wow wow, we still need this file to be existed.

Reply:
What are we using it for?

Reply:
extending our config with user one, debugging

Reply:
https://www.percona.com/blog/2020/03/23/extending-pmm-prometheus-configuration/

Reply:
But we don't have Prometheus in the container anymore...Or we use it also for something else?

Reply:
we have victoria metrics which is a replacement for prometheus.

Reply:
yes, we need this one for sure ))